packages/explorer/src/EmbeddedExplorer.ts

```diff
@@ -20,6 +20,7 @@ export interface BaseEmbeddableExplorerOptions {
     document?: string;
     variables?: JSONObject;
     headers?: Record<string, string>;
+    includeCookies?: boolean;
```

Comment:
lets add a comment here that 
`defaults to false. If you pass `handleRequest` that will override this.`